web/firefoxcom.js

```diff
@@ -303,6 +303,16 @@ class FirefoxScripting {
 }
 
 class MLManager {
+  #enabled = new Map();
+
+  constructor({ enableAltText }) {
+    this.#enabled.set("altText", enableAltText);
+  }
+
+  isEnabledFor(name) {
+    return this.#enabled.has(name);
```

Comment:
Is this correct, since it'd return `true` even if the value in the Map is falsy?
I'd probably guess that you want:
```suggestion
    return this.#enabled.get(name);
```